docs/user-guide/external.md

```diff
@@ -80,7 +80,7 @@ spec:
   external:
     link:
       switch:
-        port: # SwitchName/EthernetXXX
+        port: # SwitchName/E1/1
```

Comment:
nit: not too fond of the generic `SwitchName` followed by a specific port. I'd go with a specific switch name and port, i.e. `s5248-02/E1/1`, which would also match what we have used everywhere else, or `switchBorder` which you use further down.